### PR TITLE
[merge rule] add CPU quantization

### PR DESCRIPTION
**Pull request by jgong5**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #114994

### CODE REVIEW

**Comment by jgong5** [comment]
cc @leslie-fang-intel

File: .github/merge_rules.yaml [hunk 1]
```diff
@@ -350,6 +351,21 @@
   - Lint
   - pull
 
+- name: CPU quantization
```
Comment (jerryzh168) [inline .github/merge_rules.yaml:354]: maybe "server CPU" or "x86 CPU"

**Review by jerryzh168 — APPROVED** [review]

**Review by malfet — APPROVED** [review]

File: .github/merge_rules.yaml (hunk 1 above)
Comment (jgong5) [reply .github/merge_rules.yaml]: Thanks. Use x86 CPU now.

**Pull request closed without merging**